### DISCUSSION TITLE
Pyinstaller on OSX creates app that does nothing.

### DISCUSSION THREAD

**Discussion by johntunes** [pyinstaller/pyinstaller#7804] [PyInstaller]
I am looking for any help as to what might be wrong. My app works fine from the terminal. when I try pyinstaller with --noconsole and --windowed it creates an app which runs for a few seconds but does not display the Tinker GUI. There are 3 .png files used, that sit in the root directory of the app. also an applescript, which sits in the root. Any help would be apprecieated - I am ready to give up.

**Comment by rokm** [comment]
The frozen app probably exits due to unhandled exception. Try running the executable from the terminal, to see if you get any useful output there; assuming you are talking about generated app bundle, you can run the executable  `./dist/<name>.app/Contents/MacOS/<name>`.
